odl/solvers/functional/functional.py

```diff
@@ -1216,9 +1229,11 @@ def __init__(self, dividend, divisor):
 
         self.__dividend = dividend
         self.__divisor = divisor
-
+        
+        #HELP: Check ranges?
```

Comment:
Yes, here you would have to check the range.

Reply:
Actually, let me be a bit more specific. What needs to be checked is that the range of the two functionals are the same. Since this only inherits from `Functional`, there is no check elsewhere that will make sure that this is the case. In the same way, there is already a check that the domains are the same.